pkg/pillar/agentlog/agentlog.go

```diff
@@ -36,15 +38,17 @@ var savedPid = 0
 //          examples: logmanager
 func initImpl(agentName string, logdir string, text bool) (*os.File, error) {
 
-	var err error
 	var logf *os.File
 	if text {
-		logfile := fmt.Sprintf("%s/%s.log", logdir, agentName)
-		logf, err = os.OpenFile(logfile, os.O_RDWR|os.O_CREATE|os.O_APPEND, 0666)
+		log.SetOutput(ioutil.Discard)
+		syslogPriorities := syslog.LOG_INFO | syslog.LOG_DEBUG |
+			syslog.LOG_ERR | syslog.LOG_WARNING | syslog.LOG_NOTICE |
+			syslog.LOG_CRIT | syslog.LOG_ALERT | syslog.LOG_EMERG
+		syslogHook, err := lSyslog.NewSyslogHook("", "", syslogPriorities, agentName)
 		if err != nil {
-			return nil, err
+			log.Fatal("initImpl: Failed creating syslog hook")
```

Comment:
This will fatal is there is no reader on the other end of the syslog AF_UNIX socket, right?
That was what we saw a month ago. Did anything chnaged in the lsyslog package?

Reply:
Right, I'll introduce retries like you did before.

Reply:
Will re-introduce the retires that you did before.

Reply:
Hey @gkodali-zededa -- I don't see retries re-introduced, am I looking in the wrong place?